public/locales/he/translation.json

```diff
@@ -0,0 +1,10 @@
+{
+    "profile": "פרופיל",
+    "settings": "הגדרות",
+    "logout": "התנתק",
+    "app-name": "Expenses Dashboard",
```

Comment:
ניהול הוצאות?

Reply:
i thought it was the website name and shouldn't be translated like Facebook.